tests/scenario04.properties

```diff
@@ -0,0 +1,6 @@
+scenario.description = Happy Flow - lijst taaktypes
```

Comment:
Set description to: Happy Flow - List Taaktypes
It is preferred to use English where possible but since Taaktypes is a specific kind of object we return its name in Dutch as that is what is understood by the client.